internal/store/alert_configuration.go

```diff
@@ -40,3 +45,16 @@ func (s *Store) AlertConfigurations(projectID string, opts *atlas.ListOptions) (
 		return nil, fmt.Errorf("unsupported service: %s", s.service)
 	}
 }
+
+// CreateAlertConfiguration encapsulate the logic to manage different cloud providers
+func (s *Store) CreateAlertConfiguration(alert *atlas.AlertConfiguration) (*atlas.AlertConfiguration, error) {
+	switch s.service {
+	case config.CloudService:
+		project := alert.GroupID
+		alert.GroupID = ""
```

Comment:
can you add a `// TODO: fix me" and link to the PR where we fix this on the go client?